tests/js/spec/utils/discover/fieldRenderer.spec.jsx

```diff
@@ -133,7 +133,7 @@ describe('getFieldRenderer', function () {
     const renderer = getFieldRenderer('timestamp.to_day', {'timestamp.to_day': 'date'});
     const wrapper = mountWithTheme(renderer(data, {location, organization}));
     const text = wrapper.find('Container');
-    expect(text.text()).toEqual('September 5, 2021');
+    expect(text.text()).toEqual('Sep 5, 2021');
```

Comment:
This was an edge case within our formatting system. We always want to use the short, 3-letter month format, never the full one. This rule is now enforced in `Datetime`.